core/services/keeper/registry_interface_logs.go

```diff
@@ -42,6 +43,24 @@ func (rw *RegistryWrapper) GetLogListenerOpts(minIncomingConfirmations uint32, u
 			},
 			MinIncomingConfirmations: minIncomingConfirmations,
 		}, nil
+	case RegistryVersion_1_3:
+		return &log.ListenerOpts{
+			Contract: rw.contract1_3.Address(),
+			ParseLog: rw.contract1_3.ParseLog,
+			LogsWithTopics: map[common.Hash][][]log.Topic{
+				registry1_3.KeeperRegistryKeepersUpdated{}.Topic():         nil,
+				registry1_3.KeeperRegistryConfigSet{}.Topic():              nil,
+				registry1_3.KeeperRegistryUpkeepCanceled{}.Topic():         nil,
+				registry1_3.KeeperRegistryUpkeepRegistered{}.Topic():       nil,
+				registry1_3.KeeperRegistryUpkeepPerformed{}.Topic():        upkeepPerformedFilter,
+				registry1_3.KeeperRegistryUpkeepGasLimitSet{}.Topic():      nil,
+				registry1_3.KeeperRegistryUpkeepMigrated{}.Topic():         nil,
+				registry1_3.KeeperRegistryUpkeepReceived{}.Topic():         nil,
+				registry1_3.KeeperRegistryUpkeepPaused{}.Topic():           nil,
```

Comment:
unpause

Reply:
Done. I think log listener does not affect mailroom delivery and retrieval.

Reply:
hmm yeah, this would be covered by integration test which we should add as a followup